blogposts/2021/api-documentation-generated-for-all-your-code.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Sneak peek: API documentation generated for all your code"
+description: "Here's a sneak peek of our latest experiment to bring API documentation to all your code, generated by LSIF code intelligence data."
+author: Stephen Gutekanst
+authorUrl: 
+publishDate: 2021-07-21T10:00-07:00
+tags: [blog, code, search, software, engineering, testing]
+slug: api-documentation-for-all-your-code
+heroImage: https://sourcegraphstatic.com/blog/api-docs/sourcegraph-api-docs.png
+socialImage: https://sourcegraphstatic.com/blog/api-docs/sourcegraph-api-docs.png
+published: true
+---
+
+At Sourcegraph, we have a small team working hard on a new experimental feature: API docs.
+
+In order to build awesome code search we've built a global graph of code that has a deep, semantic understanding of the underlying code—which we represent as [LSIF code intelligence data](https://lsif.dev). Using Sourcegraph's LSIF code intelligence data, we can analyze your code and generate API documentation for all of it. We've started with Go code and intend to expand to more languages as we develop this feature. 
+
+![API Docs](https://sourcegraphstatic.com/blog/api-docs/api-docs.png)
```

Comment:
@slimsag Sorry I forgot to mention this before, could you please add captions to the screengrabs? If someone is skimming the post we have a better chance of drawing them in if they can see at a glance what an image or gif is about. You can see some examples here: https://about.sourcegraph.com/blog/optimizing-a-code-intel-commit-graph-part-2/

Also, the images aren't consistent sizes (the feedback widget one is comically large 😆) and I think the only way to apply sizing to them is with HTML. This is not super familiar for me but if you know what you're doing please go ahead and make them consistent!

Reply:
Done, I updated all locations! As for the HTML stuff, in case you're curious to learn - the trick is just to write the image like this instead:

```html
<img height="50px" src="https://sourcegraphstatic.com/blog/api-docs/api-docs.png"></img>
```

(and if you just set `height`, the image width will be automatically adjusted)